### ISSUE TITLE
Help with integration with Arch Update Checker

### ISSUE THREAD

**Issue by dhruv8sh** [eworm-de/pacman-offline#4]
TL;DR: Help with tracking package download progress to integrate with Arch Update Checker

I am the owner of the [Arch Update Checker](https://github.com/dhruv8sh/arch-update-checker) applet on KDE Plasma. 
Recently, a user asked for integration with your script [here](https://github.com/dhruv8sh/arch-update-checker/issues/54). I think it is a great idea and thought it would be really good to ask for your help with it.
I would need help with tracking progress/which packages have been downloaded, since it is difficult to track them inside the caches.
I was thinking if your script is downloading packages in the cache. We could change the cache location to something else. Download packages there, update the system and move the caches back to original cache location. Optionally, of course. This would make it easier to track them.
Thanks a lot.

**Comment by eworm-de** [comment]
I think `pacman-offline` should integrate with `pacman` as close as possible, without changing paths or what ever. It would break my use cases, and even if optional it would probably cause more trouble than it helps.

However... If you really want this... All I do to download packages is running `pacman -Suw`. You could do that yourself, even to a different directory. When finished move the packages to the real cache directory, then run  `pacman-offline` to prepare - it will then just do some basic verification and preparation, but no downloading. This should not take longer than a few seconds.
